content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -71,7 +71,7 @@ defaulting was not compatible with `kubectl apply`. Also, once a DaemonSet is cr
 its `spec.selector` can not be mutated. Mutating the pod selector can lead to the
 unintentional orphaning of Pods, and it was found to be confusing to users.
 
-The `spec.selector` is an object consisting of two fields:
+The `.spec.selector` is an object consisting of two fields:
```

Comment:
Good catch. Can you please fix up the one on l71 too? (I gave a quick look through, but you might want to check one more time in case there are any others.)

Reply:
fixed